.github/workflows/continuous-integration.yaml

```diff
@@ -1002,6 +1037,15 @@ jobs:
           ec2-instance-id: ${{ needs.start-runner-linux.outputs.ec2-instance-id-310 }}
           mode: stop
 
+      - name: Stop EC2 runner python 3.11
+        uses: machulav/ec2-github-runner@fcfb31a5760dad1314a64a0e172b78ec6fc8a17e
+        if: ${{ always() && needs.start-runner-linux.outputs.ec2-instance-id-311 }}
+        with:
+          github-token: ${{ secrets.EC2_RUNNER_BOT_TOKEN }}
+          label: ${{ needs.start-runner-linux.outputs.label-311 }}
+          ec2-instance-id: ${{ needs.start-runner-linux.outputs.ec2-instance-id-311 }}
+          mode: stop
```

Comment:
hopefully this will not break @RomanBredehoft 's green weekly build

Reply:
we probably need to update our GH secrets here

Reply:
Looks like we don't